### PR TITLE
Bug 1225418 - Port Download menu to use gaia-menu instead of building blocks

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1225418

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=870deb4952887e7db6150fd4bb3867da369f4421

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2d23b367670f3517765fb79db590519143f80f72

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=39be65bf5d2f4c1c8ed1f8144a79503113190e1d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0bcc91fd918a1e7e5253f453cce61f35135b67dc

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=74a0f07a9839c9a431254cf360ef1b2899a3a560

File: shared/js/download/download_ui.js [hunk 1]
```diff
@@ -224,16 +223,14 @@ var DownloadUI = (function() {
   }
 
   function addActionMenu() {
+    // If we have an actionMenu, empty it and re-create it.
     if (actionMenu !== null) {
       actionMenu.innerHTML = '';
```
Comment (nullaus) [inline shared/js/download/download_ui.js]: I think if you're removing the actionMenu, there's no need to set innerHTML empty.

File: shared/js/download/download_ui.js [hunk 2]
```diff
@@ -242,7 +239,9 @@ var DownloadUI = (function() {
     }
 
     actionMenu.innerHTML = '';
```
Comment (nullaus) [inline shared/js/download/download_ui.js]: Ditto, since you're removing it this can go away.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d15045b9f9f13098890edb5d91541ff04306ae89

**Pull request merged**